src/index.js

```diff
@@ -1,18 +1,86 @@
 const { Telegraf } = require("telegraf");
 require("dotenv").config();
+const fs = require("fs");
+const path = require("path");
 const Groq = require("groq-sdk");
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
 const groq = new Groq();
 
-bot.start((ctx) => ctx.reply("Witaj, hehe, jestem Seba, jak mogę Ci pomóc?"));
+const memoryFile = path.join(__dirname, "memory.json");
 
-// Handle messages and respond using Groq SDK
+// Tworzenie pliku pamięci, jeśli nie istnieje
+if (!fs.existsSync(memoryFile)) {
+  fs.writeFileSync(memoryFile, JSON.stringify({}));
+}
+
+// Funkcja wczytania pamięci
+const loadMemory = () => {
+  try {
+    const data = fs.readFileSync(memoryFile, "utf8");
+    console.log("Memory loaded:", data);
+    return JSON.parse(data);
+  } catch (error) {
+    console.error("Error loading memory:", error);
+    return {};
+  }
+};
+
+// Funkcja zapisu pamięci
+const saveMemory = (memory) => {
+  try {
+    fs.writeFileSync(memoryFile, JSON.stringify(memory, null, 2));
```

Comment:
**praise**: Nice use of `JSON.stringify` with formatting for better readability of the saved memory file.